bot.py

```diff
@@ -75,10 +79,14 @@ def on_message(mosq, obj, msg):
         send_to_bot("Shutter Opened!")
     elif msg.topic == 'door/shutter/state/closed':
         send_to_bot("Shutter Closed!")
+    elif msg.topic == 'door/outer' and msg.payload == 'opened':
```

Comment:
Do we need to check the flag here?

Reply:
It doesn't make a difference unless we care about the case where the doorbell is pressed with the door open, then the door is closed :p